### PR TITLE
Classroom view almost done

### LINKED ISSUE #62: [FEATURE] - View Classroom view
### Summary
image

Design and create the View classroom view
 

### Definition of Done

When the View classroom view have been Designed and created, but still requires endpoints implementation of endpoints from backend to be fully functional

### PR DESCRIPTION
**Pull request by EMojoKrogh**
## Description

Classroom view works for instructor and student. Only thing left is to make correct redirects to related pages.

### Resolved Issue
Fixes #62 

## Changes

Classroom view created for instructors and students
Able to edit classroom and classroom status.
Able to delete classroom
Able to open and close classrooms
Students can leave classrooms.

## Additional Information

Fixed Instructor home layout issues.

## Checklist

- [ ] I have added test cases for my additions
- [x] New and old tests passed
- [ ] Documentation is updated

### CODE REVIEW

File: src/components/DropdownMenus/ProfileDropdown.svelte [hunk 1]
```diff
@@ -22,15 +22,16 @@
 	>
 	<DropdownMenu.Content>
 		<DropdownMenu.Group>
-			<DropdownMenu.Label>My Account</DropdownMenu.Label>
-			<DropdownMenu.Separator />
-			<DropdownMenu.Item class="cursor-pointer">Profile</DropdownMenu.Item>
+			<!-- <DropdownMenu.Label>My Account</DropdownMenu.Label> -->
+			<!-- <DropdownMenu.Separator /> -->
```
Comment (andreaswurtz) [inline src/components/DropdownMenus/ProfileDropdown.svelte:26]: KOMMENTARE !??!?!

File: src/components/Select/Select.svelte [hunk 2]
```diff
@@ -1,50 +1,115 @@
...
+				{#if multiple}
+					{multiple_selected.map((option) => option.label).join(', ')}
+				{:else}
+					{multiple_selected[0]?.label}
+				{/if}
+			{:else}
+				Select options
+			{/if}
+		</span>
+		<span>&#9662;</span>
```
Comment (andreaswurtz) [inline src/components/Select/Select.svelte:80]: ▼ Name: Black Down-Pointing Triangle. Unicode Version: 1.1 (June 1993)

Comment (EMojoKrogh) [reply src/components/Select/Select.svelte:80]: https://www.w3schools.com/html/html_entities.asp

File: src/components/DropdownMenus/ProfileDropdown.svelte (hunk 1 above)
Comment (EMojoKrogh) [reply src/components/DropdownMenus/ProfileDropdown.svelte]: har fixet det

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -165,3 +206,17 @@
 		</div>
 	</div>
 {/if}
+
+<style>
+	.scrollable-list {
+		overflow-y: scroll; /* Enable vertical scrolling */
+		overflow-x: hidden; /* Disable horizontal scrolling */
+		scrollbar-width: none; /* For Firefox: hide scrollbar */
+		-ms-overflow-style: none; /* For IE and Edge: hide scrollbar */
```
Comment (andreaswurtz) [inline src/routes/+page.svelte:215]: Hvad hvis jeg gerne vil have en scrollbar i internet explorer????

Comment (EMojoKrogh) [reply src/routes/+page.svelte:215]: hvis du køber en chromebook, så køber jeg en mac

**Review by andreaswurtz — APPROVED** [review]
yoghurt

**Pull request merged**